ingestion/src/metadata/utils/credentials.py

```diff
@@ -69,9 +71,9 @@ def create_credential_tmp_file(credentials: dict) -> str:
         return temp_file.name
 
 
-def build_google_credentials_dict(gcs_values: GCSValues) -> Dict[str, str]:
+def build_google_credentials_dict(gcs_values: GcsCredentialsValues) -> Dict[str, str]:
```

Comment:
Is this just a rename?

Reply:
Yes, we have changed the name of the class only.